packages/ckeditor5-emoji/src/emojirepository.ts

```diff
@@ -270,11 +270,13 @@ export default class EmojiRepository extends Plugin {
 		}
 
 		/**
-		 * By default, the Emoji plugin fetches the emoji repository from CKEditor 5 CDN. To avoid this,
-		 * you can use the {@link module:emoji/emojiconfig~EmojiConfig#definitionsUrl `emoji.definitionsUrl`}
+		 * It was detected that your installation uses a commercial license key,
+		 * and the default {@glink features/emoji#emoji-source CKEditor CDN for Emoji plugin data}.
+		 *
+		 * To avoid this, you can use the {@link module:emoji/emojiconfig~EmojiConfig#definitionsUrl `emoji.definitionsUrl`}
 		 * configuration option to provide a URL to your own emoji repository.
 		 *
-		 * If you only want to suppress this warning, set this configuration option to `cdn`.
+		 * If you want to suppress this warning, and use the default CDN, set this configuration option to `cdn`.
```

Comment:
```suggestion
		 * If you want to suppress this warning, while using the default CDN, set this configuration option to `cdn`.
```

We may consider this wording, too. It makes it clear that these two are concurrent.